src/components/NewPost/NewPost.js

```diff
@@ -4,6 +4,7 @@ import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 
 import styles from './NewPost.module.css';
+import axios from '../../axios-posts';
```

Comment:
Wiem, że nazewnictwo i to podejście wzięłaś od Maxa, ale przyznam, że używanie nazwy 'axios' w tym miejscu - ja bym to jednak zmieniła... bo to trochę sugeruje jakbyś na axiosie wykonywała akcję .post() podając tylko '/posts.json'. Jeśli ktoś się nie wczyta, skojarzy to z samym AXIOSEM: https://www.npmjs.com/package/axios, nie Twoją instancją z axios-posts. 
Zwróć też uwagę, że za chwilę sama używasz w actions.js axiosa, nie podając 'posts.json', a całą ścieżkę. Mimo, że instancja 'axiosa' ma już zdefiniowany baseURL. 
Będzie się to mylić. 
Jakakolwiek zmiana mogłaby być pomocna. Może 'axiosInstance' albo 'axiosPosts'? 
Żeby nie było - nie znalazłam informacji, żeby to była zła praktyka, itp. Widzę, że niektórzy rzeczywiście używają tej nazwy. Także nie musisz słuchać akurat tej mojej sugestii. Ale przyznam, że mi bardzo rzuca się w oczy/drażni.

Reply:
Zgadzam się, też mi to w sumie nie pasowało, a zrobiłam tak, bo Max miał tak w kursie :D zmienię na axiosInstance - tym razem też w actions.js :)